flink-formats/flink-protobuf/src/test/java/org/apache/flink/formats/protobuf/VeryBigPbProtoToRowITCase.java

```diff
@@ -25,8 +25,11 @@
 /**
  * Test for very huge proto definition, which may trigger some special optimizations such as code
  * splitting and java constant pool size optimization.
+ *
+ * <p>Implementing this test as an {@code ITCase} enables larger heap size for the test execution.
```

Comment:
```suggestion
 * <p>@implNote Implementing this test as an {@code ITCase} enables larger heap size for the test execution.
```
nit should we use `@implNote` here?

Reply:
interesting. I didn't know about it. It [seems to be](https://stackoverflow.com/a/55150289) supported by Intellij. I tried it. We have issues with checkstyle complaining about "an unknown JavaDoc tag". I tend to not use it because of that. We might have to update checkstyle first.